src/ddmd/toobj.d

```diff
@@ -75,6 +75,33 @@ extern (C++):
 alias toSymbol = ddmd.tocsym.toSymbol;
 alias toSymbol = ddmd.glue.toSymbol;
 
+private struct ClassFlags
```

Comment:
While you're at it, it wouldn't hurt to add a DDOC comment block.  I think @WalterBright  is [encouraging us to do so](https://github.com/dlang/dmd/pull/7341#discussion_r152198939) even for private code.